diff --git a/Dockerfile b/Dockerfile
index b38158f..10ee5bc 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -84,6 +84,11 @@ RUN rm -rf /var/www/*
 COPY scripts/ScriptHandler.php /var/www/scripts/ScriptHandler.php
 COPY composer.json /var/www/composer.json
 COPY composer.lock /var/www/composer.lock
+# Copy and custom modules and / or themes
+COPY html/modules/custom/ /var/www/html/modules/custom/
+COPY html/themes/custom/ /var/www/html/themes/custom/
+# Copy config directory
+COPY config/ /var/www/config/
 WORKDIR /var/www
 RUN apk --update --no-cache add git openssh-client; \
     mkdir -p /root/.ssh; echo $SSH_PRIVATE_KEY | base64 -d > /root/.ssh/id_rsa; \
